sbncode/PID/Razzle_module.cc

```diff
@@ -83,7 +83,7 @@ class Razzle : public art::EDProducer {
   art::ServiceHandle<art::TFileService> tfs;
   art::ServiceHandle<cheat::ParticleInventoryService> particleInventory;
 
-  art::InputTag fLArGeantLabel, fPFPLabel, fShowerLabel, fShowerSelVarsLabel;
+  art::InputTag fLArGeantLabel, fSimChannelLabel, fPFPLabel, fShowerLabel, fShowerSelVarsLabel;
```

Comment:
fLArGeantLabel no longer used and should be removed.

Reply:
done